lib/shopify_api/webhooks/handler.rb

```diff
@@ -4,12 +4,29 @@
 module ShopifyAPI
   module Webhooks
     module Handler
+      include Kernel
       extend T::Sig
       extend T::Helpers
-      interface!
+      abstract!
 
-      sig { abstract.params(topic: String, shop: String, body: T::Hash[String, T.untyped]).void }
+      sig do
+        params(data: T::Hash[Symbol, T.untyped]).void
```

Comment:
Should we:
- explicitly type `data`?
- make `data` an object instead of a hash to keep the API similar to what it was before? Can we use `ShopifyAPI::Webhooks::Request`?